### ISSUE TITLE
Training metric problem

### ISSUE THREAD

**Issue by Tannln** [OpenDriveLab/TopoNet#20]
Hello, thank you for your contribution to autonomous driving, I am now trying to train the code in the single card 4090. The metrics after 3 days of training are very different from your logs. Can you help me see what went wrong? Below is my training log
[20240507_104134.log](https://github.com/OpenDriveLab/TopoNet/files/15288343/20240507_104134.log)

**Comment by sephyli** [comment]
You may need to adjust learning rate according to your batch size. For your case, you should divide the lr by 8 or sqrt(8).
